frontend/views/components/group-creation-steps/GroupMincome.vue

```diff
@@ -38,21 +38,59 @@
 
       i18n.helper This value can be adjusted in the future.
 
+    fieldset.field
+      .label.c-label-tooltip
+        i18n In what day should the first payment distribution be calculated?
+        tooltip(direction='bottom-end')
+          span.button.is-icon-small
+            i.icon-question-circle
+          template(slot='tooltip')
+            i18n(tag='p') Select which day of the month should the distribution be calculated. Every group member will need to update their mincome details before this date to ensure that the algorythm can fairly distribute available funds between group members.
+
+      .selectbox
+        select.select(
+          :aria-label='L("Choose day")'
+          name='distributionDate'
+          required=''
+          :value='group.distributionDate.toString()'
```

Comment:
I don't think this should have `.toString()` as the value, as then it's not bound to the variable